src/main/java/com/vonage/quickstart/verify2/SendRequestAllChannels.java

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
This can be removed as it's not in the spec (and covered by the fallback example)